README.md

```diff
@@ -2,32 +2,32 @@ Rancher Harvester
 ========
 [![Build Status](https://drone-publish.rancher.io/api/badges/rancher/harvester/status.svg)](https://drone-publish.rancher.io/rancher/harvester)
 
-Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes.
+Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes. It is an on-prem cloud computing virtualization platform.
 
 ## Overview
 Harvester makes it easy to get HCI up and running on the bare metal servers. Here are some notable features of the Harvester that currently provided:
 1. VM lifecycle management includes SSH-Key injection, Cloud-init and, graphic and serial port console
 1. Build-in storage management based on [Longhorn](https://longhorn.io/)
```

Comment:
Same here?

Reply:
We explain what HCI means at the beginning, so thereafter we should just say HCI.